packages/typeorm/README.md

```diff
@@ -0,0 +1,92 @@
+<p align="center">
+   <br/>
+   <a href="https://next-auth.js.org" target="_blank"><img width="150px" src="https://next-auth.js.org/img/logo/logo-sm.png" /></a>
+   <h3 align="center">NextAuth.js - <b>TypeORM Adapter</b></h3>
+   <p align="center">Authentication for Next.js</p>
+   <p align="center">
+   Open Source. Full Stack. Own Your Data.
+   </p>
+   <p align="center" style="align: center;">
+      <img src="https://github.com/nextauthjs/adapters/workflows/Build%20Test/badge.svg" alt="Build Test" />
+      <img src="https://github.com/nextauthjs/adapters/workflows/Integration%20Test/badge.svg" alt="Integration Test" />
+      <img src="https://img.shields.io/bundlephobia/minzip/@next-auth/typeorm-adapter" alt="Bundle Size"/>
+      <img src="https://img.shields.io/npm/dm/@next-auth/typeorm-adapter" alt="Downloads" />
+      <img src="https://img.shields.io/github/stars/nextauthjs/adapters" alt="Github Stars" />
+      <img src="https://img.shields.io/github/v/release/nextauthjs/adapters?include_prereleases" alt="Github Release" />
+   </p>
+</p>
```

Comment:
Should we enforce a header like this on all the adapters?

Should we enforce certain sections in the README files in general?

Nonetheless we should make sure that the example adapter contains as much boilerplate for an easy start, as possible! https://github.com/nextauthjs/adapters/tree/canary/packages/example is a bit empty right now.

Reply:
Hmm yeah the headers like are probably not necessary on all the adapters. Ill see if I can whip up a simplified version.

What exactly is the goal of the example adapter actually?

Reply:
I would like to keep some basic badges for the adapters though, like 1) tests passing/failing 2) minzipped size of adapter pkg and 3) latest release version. What do you think?

Reply:
I think the goal of the example adapter should be to be able to simply copy+paste it and get your adapter working as fast as possible. All the config, known dependencies, readme, everything is filled out for you.

Reply:
We should discuss the example adapter internally first. You corrected my concerns on the `typeorm-adapter` for now.